index.rst

```diff
@@ -121,8 +121,10 @@ Welcome to the documentation for Citus 7.0! Citus horizontally scales PostgreSQL
    cloud/availability.rst
    cloud/security.rst
    cloud/scaling.rst
+   cloud/forking.rst
```

Comment:
Why this change?

Reply:
The change is not actually related to this PR per se, I was just throwing it in as a little fix.

I noticed that currently on the master branch the forking document is present but it's not listed in the menu. It was removed from the menu when we added the new front page UI (and is linked from the Cloud "portal" page in docs). I don't remember us talking about removing forking from the main menu, so it seemed like a mistake.